packages/ember-metal/lib/transaction.js

```diff
@@ -0,0 +1,75 @@
+import { meta as metaFor } from './meta';
+import { has } from 'require';
+import { deprecate, runInDebug } from 'ember-metal/debug';
+import isEnabled from 'ember-metal/features';
+
+let runInTransaction, didRender, assertNotRendered;
+
+if (isEnabled('ember-glimmer-allow-two-way-reflush')) {
+  let hasGlimmer = has('glimmer-reference');
+  let counter = 0;
+  let inTransaction = false;
```

Comment:
Should this be tied to some other state? In general, it should be possible to have two applications sharing the same ember internal modules on the page at once. The way I read the use of this module level flag, it seems that it might be an issue?